internal/watch/notify_test.go

```diff
@@ -421,9 +421,6 @@ func TestWatchNonexistentDirectory(t *testing.T) {
 	f := newNotifyFixture(t)
 	defer f.tearDown()
 
-	ignore, _ := dockerignore.NewDockerPatternMatcher(f.paths[0], []string{"./"})
```

Comment:
i'm not totally sure what these were here for, but they're totally bogus -- they prevent the fsync() in this test from working

Reply:
oh haha - now i remember, i think they're working around a bug in windows where it gets change events for the directory...let me see what i can do about that...

Reply:
fixed! addressed it the same way i fixed it for macos